### PR TITLE
Milestone 4 calculations

### PR DESCRIPTION
**Pull request by lekegitrepo**
#### As required:
- Create a calculate.js file and implement the calculate function.
  For the calculate function: various conditions to perform arithmetic operation were added.
- Create an operate.js file and implement the operate function.
  For the operate function: various cases to check arithmetic operators were added.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  switch (operation) {
+    case 'X':
+      return operandOne.times(operandTwo).toString();
+    case '-':
+      return operandOne.minus(operandTwo).toString();
+    case '+':
+      return operandOne.plus(operandTwo).toString();
+    case '%':
+      return operandOne.mod(operandTwo).toString();
```
Comment (MauricioRobayo) [inline src/logic/operate.js]: In the calculator, this is not the modulus operator, it should be a modifier turning the number into a percentage based number.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
This is looking great so far, but could you please double-check the following before continuing:

* [ ] % should be a modifier turning the number into a percentage based number, for example, 95 % == .95 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)), so we should be able to apply it even if there is only one operand:

percentage modifier

Status: __Request changes__.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  switch (operation) {
+    case 'X':
+      return operandOne.times(operandTwo).toString();
+    case '-':
+      return operandOne.minus(operandTwo).toString();
+    case '+':
+      return operandOne.plus(operandTwo).toString();
+    case '%':
+      return operandTwo.div('100').toString();
```
Comment (shloch) [inline src/logic/operate.js:17]: nice work

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    case '÷':
+      if (operandTwo === 0) return 0;
+      return operandOne.div(operandTwo).toString();
```
Comment (shloch) [inline src/logic/operate.js:20]: Manage division by ZERO in milestone 5, we'll verify that. It shouldn't return `0`, but something else. something like `NaN` or `INFINITY`....or some other creative error management you'll think about.

### Correct this in milestone 5

**Review by shloch — APPROVED** [review]
## hello @lekegitrepo 

Great job............good to  go
accomplished

**Pull request merged**